images/scorecard-test/bin/user_setup

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -x
+
+# ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
+mkdir -p "${HOME}"
+chown "${USER_UID}:0" "${HOME}"
+chmod ug+rwx "${HOME}"
+
+# runtime user will need to be able to self-insert in /etc/passwd
+chmod g+rw /etc/passwd
```

Comment:
that chmod on the passwd file is no longer needed according to Joe.

Reply:
Check https://github.com/operator-framework/operator-sdk/pull/2469 out for the full set of changes. It's more than just removing this line.